schedule/containers/engines_and_tools_docker.yaml

```diff
@@ -20,21 +20,31 @@ conditional_schedule:
         - containers/docker_runc
   supported:
     HOST_VERSION:
+      15-SP4:
+        - containers/buildah_docker
```

Comment:
I also had to add this... yet again another reason to avoid these ugly conditions if we could just have `if is_sle(>=15-SP1)` ...